### PR TITLE
Updated install instructions

### PR DESCRIPTION
**Pull request by Tech-TTGames**
## Description

An update to README.md to actually provide instructions for v0.1 installation. 

## Motivation and Context

Previous instructions are no longer valid

## Changes Made

- Update REAMDE.md to fit v0.1
- Fixed a wrong intent declaration
- Added pynacle dep to avoid warnings

## Issues Resolved

Fixed a bug.
Provided more information

## Checklist

- [X] My code follows the style guidelines of this project.
- [X] I have reviewed my own code and ensured it passes all linting tests.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] I have added tests that prove my fix is effective or that my feature works.
- [X] I have run all tests and ensured they pass.

### CODE REVIEW

**Comment by Tech-TTGames** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -43,22 +45,45 @@ Toggle with /setting stripbuttons.
...
-3. Invoke `/'venv name here'/scripts/activate` to enter into the enviroment for use.
-4. Run (Assuming pip and python are already on the machine) `pip install -r requirements.txt`
-5. Create the bot on [Discord Developers](https://discord.com/developers/applications).
+1. Open the directory you want the project to be placed in.
+2. Use `git clone https://github.com/Tech-TTGames/Tickets-Plus.git` or download and unpack [the repo](https://github.com/Tech-TTGames/Tickets-Plus/archive/refs/heads/main.zip).
+3. Ensure that python3.11 is installed and available, same for pip.
+4. Run `pip install poetry`.
+    - Change some poetry settings as-needed. You can add the `--local` flag to set those settings only to the current directory
+        - `poetry config virtualenvs.in-project true` installs the virtual environment in project not in a poetry-specific location (reccomended).
+        - `poetry config virtualenvs.create false` if you don't want poetry to use a venv (not reccomended).
```
Comment (benfoster04) [inline README.md:54]: Recommended*

File: README.md [hunk 2]
```diff
@@ -43,22 +45,45 @@ Toggle with /setting stripbuttons.
...
+    - If you want development packages add `--with dev`
+6. Install PostgreSQL. [Guide Here](https://www.postgresql.org/download/)
+    1. Set up automatic postgres startup [Linux](https://www.postgresql.org/docs/current/server-start.html) and for windows just start it via `services.msc`
+    2. Set up user and database for bot. `<FIELD>` are required and to be repalced with your own stuff. [FIELD] are optional and can be ignored.
+        - Linux:
+            1. `sudo -u postgres -i`
+            2. `createuser <dbuser> --pwprompt` The prompt will ask you for password for new user - `<dbpass>`.
+            3. `createdb <dbname> [comment] -E UTF8 -O <dbuser>` - `<dbuser>` being the same as in previous step.
+        - Windows:
+            1. If the installed postgres bin isn't in PATH use `cd` to go to the instalaltion bin.
```
Comment (benfoster04) [inline README.md:66]: Installation*

**Review by benfoster04 — CHANGES_REQUESTED** [review]
Repair typos otherwise clear to merge

**Review by benfoster04 — APPROVED** [review]

**Pull request merged**